nss-cohort-36/c36-class-website
# Jg style 2

# Description
This PR closes 2 tickets, both the editing the class fun pic and the carousel feature which scrolls at a count of 5 seconds. I also added a new NSS logo so that it looks ascetically pleasing. 

Lastly, I edited the gradient on the original class pic and the white wave via a photo editor so when our banners are scrolling via carousel, the white waves have a smooth transition. 


Closes # 
~Edit class funny photo with yearbook theme
~Implement Carousel 🎠 for class photos

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout jg-style-2`
4. Please go to http://localhost:3000/ and look ensure that the home screen presents you a banner that scrolls through at a 5 second count. 
Also, ensure that both images do show up as it should show the serious and silly picture upon loading the webpage(in that respective order)
# Checklist:
